packages/forms-web-app/src/pages/projects/router.js

```diff
@@ -25,6 +25,9 @@ const {
 const {
 	addProjectsIndexTranslationsMiddleware
 } = require('./index/_middleware/add-projects-index-translations-middleware');
+const {
+	addProjectsDocumentsTranslationsMiddleware
```

Comment:
I'm guessing this can't be behind a feature flag because the english is being picked up this way now.  Just checking.